src/mpyl/steps/deploy/kubernetes.py

```diff
@@ -64,11 +64,11 @@ def execute(self, step_input: Input) -> Output:
             if hostname:
                 has_specific_routes_configured: bool = bool(
                     builder.deployment.traefik is not None
+                    and properties.target == Target.PRODUCTION
                 )
                 self._logger.info(
                     f"Service {step_input.project.name} reachable at: {hostname}"
                 )
-
                 endpoint = (
                     "/" if has_specific_routes_configured else "/swagger/index.html"
                 )
```

Comment:
@SamTheisens I have a question about this logic as well. So currently if you have a traefik config in your service we don't attach `/swagger/index.html` to the URL which then gets put in the PR descriptions. This means that for most of our services, the link in PR description is broken. 

Just because you have traefik host overrides, the swagger documentation is going to be in the same place though right? I mean, the host can be specific for the service but the path of the swagger doc is not?

Reply:
Yeah, I think so. The default rules are always applied AFAIK. You could check in traefik if that's indeed the case. We could  re-introduce `/swagger/index.html`. Although it's very Vandebron specific...

Reply:
Do you have a better suggestion to always be able to figure out the swagger endpoint? Should we provide an option al swagger host in the project yaml?